### PR TITLE
Run helloworld server in build/target wildfly; configure build/target for ssl

### PR DESCRIPTION
**Pull request by ronsigal**

### CODE REVIEW

**Comment by ronsigal** [comment]
I've changed helloworld/service to run in the WildFly in build/target directory. Also, -Dssl= can be used to configure that WildFly for SSL connections. The possible values are "none", "oneway", and "twoway".

**Review by hpehl — COMMENTED** [review]
LGTM @ronsigal! There are some formatting issues, though.

File: examples/helloworld/service/pom.xml [hunk 1]
```diff
@@ -3,119 +3,210 @@
...
          https://www.apache.org/licenses/LICENSE-2.0
 
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
+     Unless required by applicable law or agreed   o in writing, software
+     distributed under   he License is distributed on an "AS IS" BASIS,
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
+     See   he License for   he specific language governing permissions and
```
Comment (jamezp) [inline examples/helloworld/service/pom.xml:15]: Looks like all the t's went missing :)

**Review by jamezp — CHANGES_REQUESTED** [review]
Minor nit, it looks like the POM somehow got re-formatted with 2 spaces instead of 4. While that's not a huge deal, IMO it's best for projects to stick to a standard spacing for their source files.

**Comment by hpehl** [comment]
> Minor nit, it looks like the POM somehow got re-formatted with 2 spaces instead of 4. While that's not a huge deal, IMO it's best for projects to stick to a standard spacing for their source files.

Running `./format.sh` should fix that.

**Comment by jamezp** [comment]
Is it setup to format POM's?

**Comment by hpehl** [comment]
Arrg, it misses formatting POMs right now. Going to see if this can be configured...

**Comment by hpehl** [comment]
The [`formatter-maven-plugin`](https://code.revelc.net/formatter-maven-plugin/usage.html) can only format Java source code according to a provided [Eclipse formatter config](https://github.com/wildfly-extras/wildfly-grpc-feature-pack/blob/main/build-config/src/main/resources/etc/eclipse-formatter-config.xml).

It isn't meant to format XML. So the POMs should be reformatted manually to have four space indentions again.

**Comment by jamezp** [comment]
> The [`formatter-maven-plugin`](https://code.revelc.net/formatter-maven-plugin/usage.html) can only format Java source code according to a provided [Eclipse formatter config](https://github.com/wildfly-extras/wildfly-grpc-feature-pack/blob/main/build-config/src/main/resources/etc/eclipse-formatter-config.xml).
> 
> It isn't meant to format XML. So the POMs should be reformatted manually to have four space indentions again.

It actually can do it https://code.revelc.net/formatter-maven-plugin/format-mojo.html#configXmlFile. However, it would take a separate execution and configuration to only format POM's. There's also a new feature I contributed named `<includeResources/>` to include resources in being configured.

**Comment by jamezp** [comment]
> Cool! Didn't know about that.
> 
> Does (re)formatting XML mean losing the comments, blank lines, etc.? If so, we should avoid that.

It shouldn't no. I'm working on a PR for it now :)

**Comment by ronsigal** [comment]
I just learned a couple days ago that Ctrl+Shift+F formats XML in Eclipse. It's not ideal, in the sense that it uses tabs and removes empty lines, but it's easy enough correct. The first time I changed the tabs, I changed "\ t" instead of "\t" and the initial 't's went away. ;-)

**Pull request closed without merging**